packages/json5/json5.mjs

```diff
@@ -0,0 +1,52 @@
+import { getFilenameExt } from '@nodejs-loaders/parse-filename';
+import JSON5 from 'json5';
+
+/** @typedef {import('esbuild').TransformOptions} ESBuildOptions */
+/** @typedef {`file://${string}`} FileURL */
+
+/**
+ * @type {import('node:module').ResolveHook}
+ */
+async function resolveJSON5(specifier, ctx, nextResolve) {
+	const resolved = await nextResolve(specifier);
+
+	if (getFilenameExt(specifier) !== '.json5') {
+		return resolved;
+	}
+
+	return { ...resolved, format: 'json5' };
+}
+export { resolveJSON5 as resolve };
+
+/**
+ * @type {import('node:module').LoadHook}
+ * @argument {FileURL} url
+ */
+async function loadJSON5(url, options = {}, nextLoad = async () => ({})) {
```

Comment:
```suggestion
async function loadJSON5(url, ctx, nextLoad) {
```

Node.js always give right object. So need to be pass in unit test

Reply:
Yes, this is never be missing and fields will never be missing within it.

Also, this is context, not options.